### PR TITLE
Feature/improove local installation

### PR DESCRIPTION
**Pull request by josephschito**
• Update 'yarn.lock' file
• Add 'test' environment key to 'config/database.yml.example' file
• Update 'README.md' file

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3955,14 +3955,22 @@ js-tokens@^3.0.2:
...
     argparse "^1.0.7"
     esprima "^4.0.0"
 
+js-yaml@~3.7.0:
+  version "3.7.0"
+  resolved "https://registry.yarnpkg.com/js-yaml/-/js-yaml-3.7.0.tgz#5c967ddd837a9bfdca5f2de84253abe8a1c03b80"
+  integrity sha1-XJZ93YN6m/3KXy3oQlOr6KHAO4A=
+  dependencies:
+    argparse "^1.0.7"
+    esprima "^2.6.0"
```
Comment (darmenise) [inline yarn.lock:3972]: @josephschito why did you need this downgrade? All the rest is right.

Comment (josephschito) [reply yarn.lock:3972]: I ran `yarn install --check-files`, what downgrade are you talking about?
This command just changed the `yarn.lock` file.

https://classic.yarnpkg.com/en/docs/cli/install/

**Comment by darmenise** [comment]
Fixed by other PR #87

**Comment by darmenise** [comment]
Fixed by other PR #87
